src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -87,7 +88,8 @@ export class NotebookModel extends Disposable implements INotebookModel {
 		public connectionProfile: IConnectionProfile | undefined,
 		@ILogService private readonly logService: ILogService,
 		@INotificationService private readonly notificationService: INotificationService,
-		@ITelemetryService private readonly telemetryService: ITelemetryService
+		@ITelemetryService private readonly telemetryService: ITelemetryService,
+		@IAdsTelemetryService private readonly _adstelemetryService: IAdsTelemetryService
```

Comment:
Please keep private variable names consistent within a file. In this case they aren't prefixed with _ so this should just be `adsTelemetryService`